lib/maven/parse/fromPom.ts

```diff
@@ -83,3 +130,50 @@ export function extractVersionedArtifact(n: XmldocTreeNode): VersionedArtifact &
         scope: !!scope ? scope.innerValue : undefined,
     };
 }
+
+function extractPlugin(n: XmldocTreeNode): Plugin {
+    const groupId = n.$children.find(c => c.$value.startsWith("<groupId>"));
```

Comment:
do the XML nodes get $name populated with the right thing? ... if they don't, maybe we should change that.
If they do, then they hypothetically should be available under `n.groupId` etc

Reply:
(if you can get here in the debugger, it's worth checking)